cartography/intel/aws/rds.py

```diff
@@ -105,9 +105,33 @@ def load_rds_instances(neo4j_session, data, region, current_aws_account_id, aws_
             AWS_ACCOUNT_ID=current_aws_account_id,
             aws_update_tag=aws_update_tag
         )
+        _attach_ec2_security_groups(neo4j_session, rds, aws_update_tag)
     _attach_read_replicas(neo4j_session, read_replicas, aws_update_tag)
 
 
+def _attach_ec2_security_groups(neo4j_session, instance, aws_update_tag):
+    """
+    Attach an RDS instance to its EC2SecurityGroups
+    """
+    attach_rds_to_group = """
+    MATCH (rds:RDSInstance{id:{RdsArn}}),
```

Comment:
I know we have ARN as ID but we could benefit from having this simpler for our OSS users and use the ARN attribute (assuming it's indexed)

Reply:
See https://github.com/lyft/cartography/blob/master/cartography/data/indexes.cypher#L40.

We also index on RDS' db_instance_identifier which is more of a friendly name.  I can make it clearer in the schema docs that this is the case.